charts/accurate/crds/accurate.cybozu.com_subnamespaces.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This CRD is generated automatically with controller-gen.
Could you setup a Makefile rule to update this?